common/src/main/scala/explore/components/UserSelectionForm.scala

```diff
@@ -62,50 +68,52 @@ object UserSelectionForm {
         p.supportedOrcidBrowser.map(Function.tupled(State.apply _))
       }
       .render_PS { (p, s) =>
-        Modal(
-          size = ModalSize.Large,
-          clazz = ExploreStyles.LoginBox,
-          content = ModalContent(
-            <.div(
-              ExploreStyles.LoginBoxLayout,
-              Logo(),
-              Button(
-                content =
-                  <.div(ExploreStyles.LoginOrcidButton,
-                        Image(clazz = ExploreStyles.OrcidIcon, src = WebpackResources.OrcidLogo),
-                        "Login with ORCID"
-                  ),
-                clazz = ExploreStyles.LoginBoxButton,
-                size = Big,
-                onClick = p.login >> p.message.set(none).runAsyncCB
-              ).when(s.showButtons),
-              Button(content = "Continue as Guest",
-                     size = Big,
-                     clazz = ExploreStyles.LoginBoxButton,
-                     onClick = p.guest >> p.message.set(none).runAsyncCB,
-                     icon = Icons.UserAstronaut
-              ).when(s.showButtons),
-              p.message.get.whenDefined(message =>
+        AppCtx.runWithCtx { implicit ctx =>
```

Comment:
Why do we need the ctx now if we didn't before?

Reply:
For the implicit Logger, I believe.

Reply:
> For the implicit Logger, I believe.

Yes